fixtures/ext-types/lib-with-seperate-swift-modules/uniffi.toml

```diff
@@ -0,0 +1,17 @@
+[bindings.python]
+cdylib_name = "uniffi_ext_types_lib"
+
+[bindings.kotlin]
+cdylib_name = "uniffi_ext_types_lib"
+
+[bindings.kotlin.external_packages]
+# Map the crate names from [External={name}] into Kotlin package names
+custom-types = "customtypes"
+ext-types-guid = "uniffi.ext_types_guid"
+uniffi-one = "uniffi.uniffi_one"
+
+[bindings.swift.external_modules]
+# Map the crate names from [External={name}] into Swift module names
+custom-types = "custom_types"
+ext-types-guid = "ext_types_guid"
+uniffi-one = "uniffi_one"
```

Comment:
I don't quite understand these - not only do they look like they are what the default values would be, there's no mention of the new entry documented above. The test does fail if they are removed, but I really don't yet understand why :)